apps/pm-user/src/components/molecules/index.js

```diff
@@ -2,5 +2,6 @@ export { default as Header } from './header';
 export { default as LoginForm } from './loginForm';
 export { default as MyCard } from './myCard';
 export { default as OwnProfileSummaryCard } from './ownProfileSummaryCard';
+export { default as ProfileSummery } from './profileSummery';
```

Comment:
Please check spelling... profileSummary

Reply:
Please change the spelling of the folder and scss file too.

Reply:
ok changing it